ppr-ui/src/interfaces/registration-interfaces/security-act-notice-interfaces.ts

```diff
@@ -0,0 +1,5 @@
+
+export interface AddEditSaNoticeIF {
+  noticeType: boolean
```

Comment:
Is noticeType boolean?

Reply:
No sir it is not, good catch, thank you!
Should be `SaNoticeTypes`, will fix